pytrip/vdx.py

```diff
@@ -267,8 +267,11 @@ def create_dicom(self):
         ds = FileDataset("file", {}, file_meta=meta, preamble=b"\0" * 128)
         if self.cube is not None:
             ds.PatientsName = self.cube.patient_name
+            ds.Manufacturer = self.cube.creation_info  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
         else:
-            ds.PatientsName = ""
+            ds.PatientsName = ''
+            ds.Manufacturer = ''  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
+        ds.SeriesNumber = '1'  # SeriesNumber tag 0x0020,0x0011 (type IS - Integer String)
         ds.PatientID = "123456"
         ds.PatientsSex = '0'
```

Comment:
Will be fixed with PR #258 via commit https://github.com/pytrip/pytrip/pull/258/commits/950db42f4a3a8dd62704cf441ebfb51fffce9066